decision_service/dsclientlib/ds_api_status.cc

```diff
@@ -0,0 +1,41 @@
+
+#include "ds_api_status.h"
+
+
+namespace decision_service {
+
+	int api_status::get_error_code() const {
+		return _error_code;
+	}
+
+	void api_status::set_error_code(int new_code) {
+		_error_code = new_code;
+	}
+
+	const std::string& api_status::get_error_msg() const {
```

Comment:
>std::string [](start = 7, length = 11)

Since api_status is part of our public interface, should we be exposing a char* instead of std::string?

Reply:
+1

---
In reply to: [186782757](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r186782757) [](ancestors = 186782757)